src/virtualenv/create/via_global_ref/builtin/python2/site.py

```diff
@@ -114,6 +108,45 @@ def abs_path(value):
     return value
 
 
+def translate_directory(directory):
+    if dir_starts_with(directory, exe_dir):
+        # content inside the exe folder needs to remap to original executables folder
+        orig_exe_folder = base_executable[: base_executable.rfind(sep)]
+        return "{}{}".format(orig_exe_folder, directory[len(exe_dir) :])
+    elif dir_starts_with(directory, prefix):
+        return "{}{}".format(base_prefix, directory[len(prefix) :])
+    elif dir_starts_with(directory, exec_prefix):
+        return "{}{}".format(base_exec_prefix, directory[len(exec_prefix) :])
+    return directory
+
+
+def rewrite_standard_library_sys_path():
+    """Once this site file is loaded the standard library paths have already been set, fix them up"""
+    for at, value in enumerate(sys.path):
+        value = abs_path(value)
+        # replace old sys prefix path starts with new
+        skip_rewrite = value == exe_dir  # don't fix the current executable location, notably on Windows this gets added
+        skip_rewrite = skip_rewrite  # ___SKIP_REWRITE____
+        if not skip_rewrite:
+            sys.path[at] = translate_directory(value)
+
+
+def revert_pythonpath_rewrite():
```

Comment:
I prefer python path to be written as ``python_path``

Reply:
I'm not sure it's going to make the function name clear that we're talking about `PYTHONPATH` though. You sure?

Reply:
yes

Reply:
yes